### PR TITLE
Master mass mailing new setting dli

### PR DESCRIPTION
**Pull request by dli-odoo**
task : https://www.odoo.com/web#id=29533&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
pad : https://pad.odoo.com/p/r.35235d3151dd50ddcfcddde59488a765
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mass_mailing/models/res_config.py [hunk 1]
```diff
@@ -8,9 +8,7 @@ class MassMailingConfiguration(models.TransientModel):
     _name = 'mass.mailing.config.settings'
     _inherit = 'res.config.settings'
 
-    group_mass_mailing_campaign = fields.Selection([
-        (0, "Do not organize and schedule mail campaigns (easy)"),
-        (1, "Allow using marketing campaigns (advanced)")], string="Campaigns",
-        implied_group='mass_mailing.group_mass_mailing_campaign',
-        help="""Manage mass mailign using Campaigns""")
+    group_mass_mailing_campaign = fields.Boolean(string="Mass Mailing Campaigns",implied_group='mass_mailing.group_mass_mailing_campaign',help="""Manage mass mailign using Campaigns""")
```
Comment (mdi-odoo) [inline addons/mass_mailing/models/res_config.py]: Follow the PEP 8 standard: Add space after comma.

File: addons/web/static/src/less/form_view.less [hunk 2]
```diff
@@ -502,3 +502,46 @@
...
+            button.btn-link {
+                padding: 0;
+            }
+        }
+    }
+}
+
+.oe_form_configuration h2 {
+    margin-top: 32px !important;
+}
```
Comment (mdi-odoo) [inline addons/web/static/src/less/form_view.less]: Add a new line at the end of file.

File: addons/website_mass_mailing/models/res_config.py [hunk 3]
```diff
@@ -7,8 +7,4 @@
 class MassMailingConfiguration(models.TransientModel):
     _inherit = 'mass.mailing.config.settings'
 
-    group_website_popup_on_exit = fields.Selection([
-        (0, 'Do not add extra content on website pages to encourage visitors to sign up'),
-        (1, 'Allow the use of a pop-up snippet on website to encourage visitors to sign up on a mass mailing list')
-    ], string="Website Pop-up",
-        implied_group="website_mass_mailing.group_website_popup_on_exit")
+    group_website_popup_on_exit = fields.Boolean(string="Website Popup",implied_group="website_mass_mailing.group_website_popup_on_exit")
```
Comment (mdi-odoo) [inline addons/website_mass_mailing/models/res_config.py]: Same as above.

**Comment by dli-odoo** [comment]
continue works: https://github.com/odoo/odoo/pull/14856

**Pull request closed without merging**